rke/templates/templates.go

```diff
@@ -134,7 +135,8 @@ func LoadK8sVersionedTemplates() map[string]map[string]string {
 			">=1.8.0-rancher0 <1.13.0-rancher0":        canalv18,
 		},
 		kdm.Flannel: {
-			">=1.21.0-alpha":                    flannelv0140,
+			">=1.22.0-alpha":                    flannelv0170,
```

Comment:
we don't want to change what already released v1.22.x versions use, so this would be `>=v1.22.6-rancher1-2` and `>=1.21.0-alpha <1.22.6-rancher1-2`